ui/src/main/webapp/reducer/index.js

```diff
@@ -17,9 +32,25 @@ const backendError = (state = {}, { type, err } = {}) => {
     default: return state
   }
 }
-
 export const getBackendErrors = (state) => state.get('backendError')
-export default combineReducers({ fetch, wizard, backendError, sourceWizard, home, wcpm, polling, systemUsage })
+
+const getDefaultTheme = () => (window !== window.top ? fromJS(adminTheme) : fromJS(darkTheme))
```

Comment:
I'm going to just have this use the adminTheme all the time. There's not really much of a reason to change the default theme based on whether it's in an iframe or not.